NewForce-at-Mountwest/bangazon-workforce-gnocci
# Finished Tickets

Ticket 4
https://trello.com/c/0fPVh5Q7/4-hr-should-be-able-to-edit-an-employee


## Steps to Test

Before you start: Run the SQL SEED DATA found in the .md file.
1- Press Ctrl+f5
2- Click on the Employees tab
3- You should see all Employees listed with their first name, last name, and department.
4- Click 'Edit'
5- Fill out the form then click save
6- The edited employee should show in the employee tab
7- Click on 'Edit' next to an employee
8- Fill out the form then click save
9- The edited employee should show in the employee tab

# Checklist:
- [x] I learned something
